types/module/module.go

```diff
@@ -212,6 +212,12 @@ type EndBlockAppModule interface {
 	EndBlock(sdk.Context, abci.RequestEndBlock) []abci.ValidatorUpdate
 }
 
+// CommitAppModule is an extension interface that contains information about the AppModule and Commit.
+type CommitAppModule interface {
```

Comment:
Can you briefly mention this here: https://github.com/cosmos/cosmos-sdk/blob/ccdd07ff7dbb9ae6cbe13061b38829714e4a8e2b/docs/docs/building-modules/01-module-manager.md#L38

Are we going with this name, btw or the one suggested in @yihuang's comment?

Reply:
Will do! Let me know if you think of others docs this should be added to.

I can change the name based on @yihuang's suggestion.

Reply:
The others can weight-in for suggestions, but I do like self-descriptive names as well.

For the docs, as it was required for a very specific use case, we probably do not need to mention it everywhere.
For coherence, mentioning it in the page above (`01-module-manager.md`) and the following pages may help the reader to understand what it does: https://github.com/cosmos/cosmos-sdk/blob/825245db1bc3f260e455a580db7a8c3033df850d/docs/docs/basics/00-app-anatomy.md#L110 & https://github.com/cosmos/cosmos-sdk/blob/825245db1bc3f260e455a580db7a8c3033df850d/docs/docs/basics/01-tx-lifecycle.md#L152